src/background/main.js

```diff
@@ -579,9 +579,12 @@ class Main {
       case "excludedDomains":
         return this.fxa.excludedDomains();
 
-      case "telemetry":
+      case "telemetryEvent":
         return this.telemetry.syncAddEvent(data.category, data.event);
 
+      case "telemetryScalar":
+        return this.telemetry.syncAddScalar(data.category, data.value);
```

Comment:
It's not a category, it's the scalar name minus the scalar category

Reply:
Can I call it "key" ?

Reply:
Only if you never use keyed scalars : )